darwin/dataset/local_dataset.py

```diff
@@ -10,6 +10,49 @@
 
 
 class LocalDataset(object):
+    """
+    Creates a dataset locally.
+
+    Parameters
+    ----------
+    dataset_path : Path
+        Path to the location of the dataset on the file system.
+    annotation_type : str
+        The type of annotation classes ```[tag, bounding_box, polygon]```.
+    partition : Optional[str], default: None
+        Selects one of the partitions ```[train, val, test]```.
```

Comment:
```suggestion
        Selects one of the partitions ```["train", "val", "test"]```.
```